brewpi-script/scriptlibs/pinList.py

```diff
@@ -41,26 +41,47 @@ def getPinList(boardType, shieldType):
                    {'val': 20, 'text': 'A2', 'type': 'free'},
                    {'val': 21, 'text': 'A3', 'type': 'free'}]
     elif boardType == "uno" and shieldType == "revC":
-        pinList = [{'val': 6, 'text': ' 6 (Act 1)', 'type': 'act'},
-                   {'val': 5, 'text': ' 5 (Act 2)', 'type': 'act'},
+        pinList = [{'val': 0, 'text': ' 0', 'type': 'serial'},
```

Comment:
What's with the changes to the existing revC pinlist?

Reply:
I added all pins for consistency/clarity, even those not used.  In BrewPi, "serial" or "free" do not show up as choices in the drop-downs.